eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -76,16 +75,17 @@ function Get-TocMapping {
             LogWarning "There is no service name for artifact $artifact. Please check csv of Azure/azure-sdk/_data/release/latest repo if this is intended. "
             # If no service name retrieved, print out warning message, and put it into Other page.
             $serviceName = "Other"
-            $displayName = $packageInfo[0].DisplayName.Trim()
         }
         else {
             if ($packageInfo.Length -gt 1) {
                 LogWarning "There are more than 1 packages fetched out for artifact $artifact. Please check csv of Azure/azure-sdk/_data/release/latest repo if this is intended. "
             }
             $serviceName = $packageInfo[0].ServiceName.Trim()
-            $displayName = $packageInfo[0].DisplayName.Trim()
         }
-        $orderServiceMapping[$artifact] = @($serviceName, $displayName)
+        $displayName = $packageInfo[0].DisplayName.Trim()
+        $isTrack2 = $packageInfo[0].New -eq 'true'
+        $isClient = $packageInfo[0].Type -eq 'client'
+        $orderServiceMapping[$artifact] = @($isClient, $isTrack2, $serviceName, $displayName)
```

Comment:
Lets sort by new then client so that we also get the track 2 mgmt higher then the track 1 client or mgmt libraries.

Reply:
```suggestion
        $orderServiceMapping[$artifact] = [PSCustomObject][ordered]@{
          New = $packageInfo[0].New
          Type = $packageInfo[0].Type
          ServiceName = $serviceName
          DisplayName =$packageInfo[0].DisplayName.Trim()
       }
```